doc/index.rst

```diff
@@ -39,7 +39,8 @@ This package can be installed through `pip` using:
 
 .. code-block::
 
-    $ pip install benchopt
+    $ git clone https://github.com/benchopt/benchOpt
+    $ pip install -e benchOpt
```

Comment:
don't assume people have git on their machine:

use:

`pip install -U https://github.com/benchopt/benchOpt/archive/master.zip`

besides I would suggest the 2 version. If you want to install the last release use this.... To install the latest development version use this...

thanks

Reply:
One might argue that `git` is used in several other section of the doc when cloning benchmarks.

Nonetheless, in e676220 I added the two possibility (last release and latest dev version). I also added a note (to be removed after the next release) recommending to use the development version at the moment (because of the discrepancy between the doc and PyPI version).

Reply:
Regarding the `git` question, it could  be a requirement, and we could write benchopt commands to wrap `git clone` and `git pull` to download and update standard benchmarks.